src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -2029,7 +2147,12 @@ private TokenFlags ScanStringExpandable(StringBuilder sb, StringBuilder formatSb
                     if (c1 != 0)
                     {
                         SkipChar();
-                        c = Backtick(c1);
+                        string str = Backtick(c1);
+                        if (IsSurrogatePair(str, ref c)) //, sb, formatSb))
```

Comment:
I'm an idiot.  Will undo the temp comment (was testing for no sb passed into method).